releases/release-1.12/release_team.md

```diff
@@ -1,13 +1,13 @@
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
-| Lead |||
-| Features | Stephen Augustus ([@justaugustus](https://github.com/justaugustus)) ||
-| CI Signal |||
-| Test Infra |||
+| Lead | Tim Pepper ([@tpepper](https://github.com/tpepper)) | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) |
+| Features | Stephen Augustus ([@justaugustus](https://github.com/justaugustus)) | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) |
+| CI Signal || Christian Hernandez ([@christianh814](https://github.com/christianh814)) |
+| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)) |
 | Bug Triage |||
 | Branch Manager |||
-| Docs |||
+| Docs | Zach Arnold ([@zparnold](https://github.com/zparnold)) | Samuel Tauil ([@samueltauil](https://github.com/samueltauil)) |
 | Release Notes |||
-| Communications |||
+| Communications | Kaitlyn Barnard ([@kbarnard10](https://github.com/kbarnard10)) ||
 | Patch Release Manager |||
```

Comment:
I'd like to be the Patch Release Manager

Reply:
Sounds good, @feiskyer. Thanks for volunteering!